### ISSUE TITLE
Define custom loss function

### ISSUE THREAD

**Issue by zouqian2468** [microsoft/LightGBM#3138]
How can I define custom loss function on the scala version of lightGBM?

**Comment by StrikerRUS** [comment]
Sorry, we do not maintain scala language here. Please ask your question in the repo from where you've got bindings for scala language (is it MMLSpark?).
